csharp_dotnetcore/Using-Cards/Using_Cards/BotBuilder.ruleset

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Rules for StyleCop.Analyzers" Description="Code analysis rules for StyleCop.Analyzers.csproj." ToolsVersion="14.0">
+  <Rules AnalyzerId="AsyncUsageAnalyzers" RuleNamespace="AsyncUsageAnalyzers">
+    <Rule Id="UseConfigureAwait" Action="Warning" />
+    <Rule Id="AvoidAsyncVoid" Action="Warning" />
+  </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp.Features" RuleNamespace="Microsoft.CodeAnalysis.CSharp.Features">
+    <Rule Id="IDE0003" Action="None" />
+  </Rules>
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <Rule Id="SA1101" Action="None" /> <!-- local calls prefixed with this -->       
+    <Rule Id="SA1129" Action="None" /> <!-- don't use value type constructor-->
+    <Rule Id="SA1200" Action="None" />
+    <Rule Id="SA1214" Action ="None"/>    <!-- encapsalate field in property-->
```

Comment:
Spelling. encapsalate => encapsulate. 

(Note: This is probably originally my type, which is amusing. But it needs to be fixed everywhere.

Reply:
Also, please remove the space before "="